backend/sirius-web-graphql/pom.xml

```diff
@@ -89,6 +89,11 @@
 			<artifactId>sirius-web-collaborative-trees-api</artifactId>
 			<version>0.3.3</version>
 		</dependency>
+		<dependency>
+			<groupId>org.eclipse.sirius.web</groupId>
+			<artifactId>sirius-web-collaborative-validation-api</artifactId>
+			<version>0.3.3</version>
+		</dependency>
```

Comment:
This project depends on sirius-web-graphql-schema and this dependency has been added there already

Reply:
That means the three dependencies above should be removed too. Not necessary in this PR.

Reply:
Done